tapiro/core/src/main/scala/io/buildo/tapiro/Util.scala

```diff
@@ -71,11 +90,18 @@ class Util() {
         controllersRoutes.foreach {
           case ((controllerType, pathName), routes) =>
             val controllerName = typeNameString(controllerType)
+            val authTypeString = Meta.authTypeString(controllerType)
```

Comment:
Maybe we can add `.getOrElse("AuthToken")` here and remove it below? (I think it's done on all occurrences below)